client/me/concierge/main.js

```diff
@@ -1,13 +1,42 @@
+/** @format */
+
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
 
-const ConciergeMain = () => {
-	return (
-		<div>{ 'Concierge chats page placeholder.' }</div>
-	);
-};
+/**
+ * Internal dependencies
+ */
+import { fetchConciergeShifts } from 'state/concierge/actions';
+import { getConciergeShifts, isFetchingConciergeShifts } from 'state/selectors';
+
+class ConciergeMain extends Component {
+	componentDidMount() {
+		// TODO: pass in the real WP.com concierge schedule id.
+		this.props.fetchConciergeShifts( 123 );
+	}
+
+	// TODO: render for real
+	render() {
+		const { isFetching, shifts } = this.props;
+
+		if ( isFetching ) {
+			return <div>{ 'Fetching ... please wait.' }</div>;
+		}
+
+		return <div>{ JSON.stringify( shifts ) }</div>;
```

Comment:
Here for temporary display purposes?

Reply:
Yes :)
To avoid future confusion, I've moved the TODO comment closer.